packages/bitflag-js/test/bitfield.test.ts

```diff
@@ -0,0 +1,176 @@
+import { describe, expect, test } from "vitest";
+import type { BitFlags } from "../src/bitfield.js";
+import { BitField } from "../src/bitfield.js";
+
+const Flags: BitFlags = {
+	FIRST: 1n << 1n,
+	SECOND: 1n << 2n,
+	THIRD: 1n << 3n,
+	FOURTH: 1n << 4n,
+	FIFTH: 1n << 5n,
+};
+
+describe("resolve", () => {
+	test("Handles numbers", () => {
+		expect(BitField.resolve(Flags.FIRST)).toEqual(Flags.FIRST);
+		expect(BitField.resolve(Flags.FIFTH)).toEqual(Flags.FIFTH);
```

Comment:
Are these actually testing plain numbers? Flags.FIRST and Flags.FIFTH are both bigints I think.